docker-registry/README.md

```diff
@@ -0,0 +1,27 @@
+Docker-registry
+===============
+
+```
+docker run -d \
+         --name ceph-docker-registry
+         -e AWS_BUCKET=mybucket \
+         -e AWS_KEY=myawskey \
+         -e AWS_SECRET=myawssecret \
+         -e AWS_HOST=myowns3.com \
+         -e AWS_SECURE=true \
+         -e AWS_PORT=80 \
+         -e AWS_CALLING_FORMAT=boto.s3.connection.OrdinaryCallingFormat \
```

Comment:
default is empty, most cases people configure the wildcard DNS
I put it only because it is worth mention, specially for automations not dependent on DNS